Frontend/android-native/app/build.gradle

```diff
@@ -40,13 +40,13 @@ repositories {
 }
 
 android {
-    compileSdkVersion 31
+    compileSdkVersion 33
```

Comment:
@ashkriti Is it necessarily required to upgrade

Reply:
Yes